doc/changelog.md

```diff
@@ -157,78 +293,72 @@
   [{'expval0': (array(-0.0978434), array(-0.19767681))}, (array(-0.0978434), array(-0.19767681))]
   ```
 
-* Add native support for `qml.PSWAP` and `qml.ISWAP` gates on Amazon Braket devices. Specifically, a circuit like
-
-  ```py
-  import pennylane as qml
-  from catalyst import qjit
+* Support has been added for linear algebra functions which depend on computing the eigenvalues
+  of symmetric matrices, such as `np.sqrt_matrix()`.
+  [(#488)](https://github.com/PennyLaneAI/catalyst/pull/488)
 
-  dev = qml.device("braket.local.qubit", wires=2, shots=100)
+  For example, you can compile `qml.math.sqrt_matrix`:
 
-  @qjit
-  @qml.qnode(dev)
-  def f(x: float):
-      qml.Hadamard(0)
-      qml.PSWAP(x, wires=[0, 1])
-      qml.ISWAP(wires=[1, 0])
-      return qml.probs()
+  ```python
+  @qml.qjit
+  def workflow(A):
+      B = qml.math.sqrt_matrix(A)
+      return B @ A
   ```
```

Comment:
@maliasadi I am a bit confused here, because A seems to be a NumPy array (and so `qml.math.sqrt_matrix()` would dispatch to `np.sqrt_matrix()`. That would imply it is run at tracing time, rather than being captured by JAX/JAXPR?

Reply:
@josh146 This would lower to `custom_calls` only for matrix variables. If the matrix is known at compile time, `qml.math.sqrt_matrix(A)` can be calculated while tracing. however, for variable matrices (like this example, `A` is a run time parameter), JAX lowers the code to `custom_call` LAPACK functions.

Reply:
@maliasadi so this requires that `A` is a JAX array/runtime parameter?

Reply:
got it

Reply:
yes!